data/json/items/melee/swords_and_blades.json

```diff
@@ -286,6 +322,24 @@
     "qualities": [ [ "CUT", 1 ], [ "BUTCHER", 15 ] ],
     "flags": [ "DURABLE_MELEE", "SHEATH_SWORD" ]
   },
+  {
+    "id": "machete_gimmick",
+    "name": { "str": "machete multitool" },
+    "type": "TOOL",
+    "description": "A thin, wobbly steel blade with sawteeth on one side and a flat chisel tip for digging. A jack of many trades and a master of none.",
+    "symbol": "/",
+    "color": "green",
+    "weight": 522,
+    "volume": 1,
+    "price": 2300,
+    "price_postapoc": 1800,
+    "to_hit": -1,
```

Comment:
base: -2
grip: weapon +2
length: short +1
surface: line +1
balance: neutral +0 or balanced +1
should be +1 or +2

bashing damage should be 8, cutting damage about 15.

Reply:
@xanderrootslayer I see bashing 8 and cutting 15 for the "machete_gimmick" now - can you confirm and mark this as resolved?

Reply:
fix'd also